kinto/core/storage/postgresql/__init__.py

```diff
@@ -654,6 +655,9 @@ def _format_conditions(self, filters, id_field, modified_field,
                 if len(value) == 0:
                     value = (None,)
 
+            if filtr.operator == COMPARISON.LIKE:
+                value = '%{0}%'.format(value)
```

Comment:
Yes I will be enough and tested for all backends. Including the `kinto-redis` community one.